gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/__init__.py.j2

```diff
@@ -13,6 +15,16 @@ from .grpc_asyncio import {{ service.name }}GrpcAsyncIOTransport
 {% if 'rest' in opts.transport %}
 from .rest import {{ service.name }}RestTransport
 from .rest import {{ service.name }}RestInterceptor
+{% if rest_async_io_enabled %}
+ASYNC_REST_CLASSES: Tuple[str, ...]
+try:
+    from .rest_asyncio import Async{{ service.name }}RestTransport
+    ASYNC_REST_CLASSES = ('Async{{ service.name }}RestTransport',)
```

Comment:
This will have more than one values once we implement interceptors. See related PR: https://github.com/googleapis/gapic-generator-python/pull/2151/files#diff-1071e950a4102dce85de9f8cbf60cd088e8e12f28877b55bd55ec4089b859ab4